pdc/apps/repository/models.py

```diff
@@ -37,6 +37,7 @@ def export(self):
 class ContentFormat(models.Model):
     # rpm, kickstart, iso
     name          = models.CharField(max_length=50, unique=True)
+    pdc_endpoint = models.CharField(max_length=200, null=True)
```

Comment:
Updating this field can invalidate some `file_primary_key` values.

Reply:
Yes, this value can be changed in admin page. not from UI or rest api.
And I think this could be set once. 
And if it's changed, all the values will changed in release_files-> repo->content-format->pdc_endpoint, but the content format type can't be changed, as it is in repo.
It's not a problem, so what do your mean?

Reply:
Seems the purpose of `pdc_endpoint` field only to tell user where to find the data for `file_primary_key`. It's not used in code. think it's better not to store `pdc_endpoint` in DB, just serialize it as an derived value - code already knows which endpoint to use for `file_primary_key`. Is that correct?

Reply:
I assume it's not, even the first people who inserted the data know the endpoint, but maybe the other guys get the data not know the endpoint.
We keep the endpoint here, that can make sure the api uris are correct when ct is using.

BTW: Have moved the pdc_endpoint to the top level.

Reply:
Currently `pdc_endpoint` for all content types is null. Updating the value will require new migrations or manual action to set the values. This is unnecessary complication since the code already knows which data model to use for which content type. I.e. it's better not to store `pdc_endpoint` in DB.

Reply:
For this column, I have discussed with Ren and asked for the reasons before, actually he thinks this needs to store in db :).
I think adding it here is for that people can get the official endpoint name from pdc, from Ren's view, it's an important thing I think :)
And it's easy to add the value from admin although currently we just need the rpms.